.prettierrc

```
@@ -0,0 +1,14 @@
+{
+  "overrides": [
+    {
+      "files": ["{**/*,*}.{js,ts,jsx,tsx}"],
```

```suggestion
      "files": ["{**/*,*}.{js}"],
```
`ts,jsx,tsx` useless in this project